# This is a gimbal control message
float64 enemy_dist
float64 enemy_yaw
float64 enemy_pitch